go.mod

```diff
@@ -96,7 +98,7 @@ require (
 	k8s.io/klog/v2 v2.120.1
 	k8s.io/kube-aggregator v0.28.15
 	k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9
-	k8s.io/kubernetes v1.28.15
+	k8s.io/kubernetes v1.29.14
```

Comment:
It would be best to keep this dependency in the Kubernetes v1.28 release to avoid any potential compatibility issues with older v0.28.15 k8s libraries.